backend/homework/7/src/main/java/com/example/config/Application.java

```diff
@@ -0,0 +1,116 @@
+package com.example.config;
+
+import com.example.interfaces.VehicleService;
+import com.example.inventory.VehicleInventory;
+import com.example.logging.Logging;
+import com.example.model.Vehicle;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = "com.example")
+public class Application {
+
+    private final VehicleService factory1Service;
+    private final VehicleService factory2Service;
+    private final VehicleInventory vehicleInventoryFactory1;
+    private final VehicleInventory vehicleInventoryFactory2;
+
+    @Autowired
+    public Application(
+            @Qualifier("factory1") VehicleService factory1Service,
+            @Qualifier("factory2") VehicleService factory2Service,
+            @Qualifier("factory1Inventory") VehicleInventory vehicleInventoryFactory1,
+            @Qualifier("factory2Inventory") VehicleInventory vehicleInventoryFactory2) {
+        this.factory1Service = factory1Service;
+        this.factory2Service = factory2Service;
+        this.vehicleInventoryFactory1 = vehicleInventoryFactory1;
+        this.vehicleInventoryFactory2 = vehicleInventoryFactory2;
+    }
+
+    @PostConstruct
+    public void printVehicleDetails() {
+        // Print Factory1 details
+        printFactoryDetails(factory1Service, vehicleInventoryFactory1, "Factory1");
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.